bootstrap/config.json

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
This is our config for the customized version of Bootstrap we have. Basically it's just typography, the grid, forms, buttons, glyphicons, and panels.